```
ERROR: test suite for <class 'casexml.apps.phone.tests.test_state_hash.StateHashTest'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_state_hash.py", line 29, in setUpClass
    cls.user = create_restore_user(domain=cls.project.name)
  File "corehq/ex-submodules/casexml/apps/phone/tests/utils.py", line 35, in create_restore_user
    'something': 'arbitrary'
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1470, in create
    commcare_user.save(**get_safe_write_kwargs())
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1415, in save
    super(CommCareUser, self).save(**params)
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1277, in save
    raise self.Inconsistent("CouchUser with username %s already exists" % self.username)
Inconsistent: CouchUser with username mclovin already exists
```

--- a/corehq/ex-submodules/casexml/apps/phone/tests/test_state_hash.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/test_state_hash.py
@@ -24,6 +24,7 @@ class StateHashTest(TestCase):
     
     @classmethod
     def setUpClass(cls):
+        super(StateHashTest, cls).setUpClass()
         delete_all_users()
         cls.project = Domain(name='state-hash-tests-project')
         cls.project.save()
@@ -42,6 +43,7 @@ class StateHashTest(TestCase):
     def tearDownClass(cls):
         cls.project.delete()
         delete_all_users()
+        super(StateHashTest, cls).tearDownClass()
 
     @run_with_all_backends
     def testEmpty(self):

--- a/corehq/ex-submodules/casexml/apps/phone/tests/test_state_hash.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/test_state_hash.py
@@ -24,6 +24,7 @@ class StateHashTest(TestCase):
     
     @classmethod
     def setUpClass(cls):
+        delete_all_users()
         cls.project = Domain(name='state-hash-tests-project')
         cls.project.save()
         cls.user = create_restore_user(domain=cls.project.name)

